site/app/templates/polls/AllPollsPageInstructor.twig

```diff
@@ -1,25 +1,30 @@
 <div class="content">
+    <input type="hidden" id="csrf_token" value="{{ csrf_token }}"/>
```

Comment:
There should be a globally defined `csrfToken` JS variable you can use instead of passing it in again here. You can do a console.log to confirm